stock_account_move_reset_to_draft/models/account_move.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+from odoo.exceptions import UserError
+from odoo.tools.float_utils import float_is_zero
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def button_draft(self):
+        """If it is a purchase invoice, we will create a new SVL for each line with
+        the sum of the value in opposite sign.
+        """
+        for item in self.sudo().filtered(
+            lambda x: x.is_inbound
```

Comment:
```suggestion
            lambda x: x.is_inbound()
```
`is_inbound` being a function it would always evaluate to True if not called.

Reply:
Indeed, good catch!

Moreover `is_inbound` was not the right method as that's based on the money flow.
The correct one is `is_purchase_document`, so I changed it